test/base32min-test.coffee

```diff
@@ -0,0 +1,76 @@
+vows = require 'vows'
+assert = require 'assert'
+base32 = require '../dist/base32.min.js'
+crypto = require 'crypto'
+
+suite = vows.describe 'Base32 Encoding'
```

Comment:
We should really not duplicate the actual code for the tests, and instead have one include for the shared suites, and then run separate tests over the unminified and minified version.

But I guess that can happen in a later PR :-)